openai copy.py

```diff
@@ -1,185 +1,111 @@
 import os

```

Comment:
Key areas needing improvement:
1. Avoid hard-coded TOTP secrets in the code for better security, and ensure proper exception handling to cover WebDriver initialization.

Example issue:
```python
+    totp = pyotp.TOTP("vwf7p7vlrwgkq6q7")  # Replace with your TOTP secret
```

Ensure these issues are addressed promptly for improved security and robustness.